src/Sylius/Bundle/ShopBundle/Resources/config/routing/ajax.yml

```diff
@@ -9,6 +9,12 @@ sylius_shop_ajax_cart:
     resource: "@SyliusShopBundle/Resources/config/routing/ajax/cart.yml"
     prefix: /cart
 
+sylius_shop_ajax_shipping_fee:
+    path: /shipping-fee/{shippingMethodId}
```

Comment:
Better use the code. Usage of IDs in front-end is generally avoided.

Reply:
Fixed